public/index.html

```diff
@@ -215,21 +218,21 @@ <h2 id="summers">Open water</h2>
+            <h3 class="h3" id="jim">New coastal permafrost research shows unexpected results</h3>
+            <p>A new publication, co-authored by BLE LTER investigator James McClelland, uses subsurface mapping to show permafrost to be mostly absent under the shallow seafloor in Kaktovik Lagoon. This research poses implications on sea-land water exchange, carbon dynamics, and coastal erosion. Read <a href="https://advances.sciencemag.org/content/6/43/eabb5083" target="_blank" aria-label="go to publication">the paper on Science Advances</a> or <a href="https://www.jsg.utexas.edu/news/2020/10/coastal-permafrost-more-susceptible-to-climate-change-than-previously-thought/" target="_blank" aria-label="go to publication">coverage</a> by The University of Texas Jackson School of Geosciences.</p>
             <br>
-            <h2 class="h3" id="brian">Graduate student spotlight: Brian Kim</h2>
-            <p>The LTER Network Office recently interviewed <a href="student.html#brian">Brian Kim</a> as part of an ongoing LTER graduate student spotlight series. Alongside his research in the sediment biogeochemistry of the lagoons, Brian also serves as the LTER-wide graduate student representative and runs <a href="https://www.instagram.com/lter_community/" target="_blank">an Instagram account</a> documenting student activities. Read <a href="https://lternet.edu/stories/lter-graduate-student-spotlight-brian-kim/" target="_blank">his interview here</a> to learn about his research, experience in graduate school, and stories about field work with the BLE team.</p>
+            <h3 class="h3" id="brian">Graduate student spotlight: Brian Kim</h3>
+            <p>The LTER Network Office recently interviewed <a href="student.html#brian">Brian Kim</a> as part of an ongoing LTER graduate student spotlight series. Alongside his research in the sediment biogeochemistry of the lagoons, Brian also serves as the LTER-wide graduate student representative and runs <a href="https://www.instagram.com/lter_community/" target="_blank" aria-label="go to Instagram page">an Instagram account</a> documenting student activities. Read <a href="https://lternet.edu/stories/lter-graduate-student-spotlight-brian-kim/" target="_blank" aria-label="go to publication">his interview here</a> to learn about his research, experience in graduate school, and stories about field work with the BLE team.</p>
```

Comment:
labels should include "in new tab", right?

Reply:
fixed in c4d830a7a3cf15b16021792b9e4da7d42427b8c5